webapp/src/main/java/com/box/l10n/mojito/service/ai/openai/OpenAILLMService.java

```diff
@@ -190,15 +310,68 @@ private AICheckResult parseResponse(
     return result;
   }
 
-  private static String getFormattedPrompt(String prompt, String sourceString, String comment) {
-    String systemPrompt = "";
+  private static String getStringChecksFormattedPrompt(
+      String prompt, String sourceString, String comment) {
+    String formattedPrompt = "";
     if (prompt != null) {
-      systemPrompt =
+      formattedPrompt =
           prompt
               .replace(SOURCE_STRING_PLACEHOLDER, sourceString)
               .replace(COMMENT_STRING_PLACEHOLDER, comment);
     }
-    return systemPrompt;
+    return formattedPrompt;
+  }
+
+  protected String getTranslationFormattedPrompt(
```

Comment:
Should we have some tests with examples for getTranslationFormattedPrompt() and prompt formatting in general?

Reply:
There is some tests in `OpenAILLMServiceTest` for getTranslationFormattedPrompt() that tests the templating and verifies the output of that function is as expected